simulation/core/sim_dataset.py

```diff
@@ -125,7 +128,8 @@ def __init__(self,
                  shuffle_block_size: Optional[int] = None,
                  sampling_method: str = 'balanced',
                  sampling_granularity: int = 1,
-                 batching_method: str = 'random') -> None:
+                 batching_method: str = 'random',
+                 allow_unsafe_types: bool = False) -> None:
```

Comment:
Once this goes in, can you also submit a PR to foundry that upstreams this arg to there? I don't think diffusion repo needs it because it just passes in **kwargs, but foundry datasets don't do that.